src/test/java/teammates/common/datatransfer/questions/FeedbackRankQuestionDetailsTest.java

```diff
@@ -0,0 +1,44 @@
+package teammates.common.datatransfer.questions;
+
+import org.testng.annotations.Test;
+
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@link FeedbackRankQuestionDetails}.
+ */
+public class FeedbackRankQuestionDetailsTest extends BaseTestCase {
```

Comment:
add a test for setting/getting duplicates too + you could also test the the default value is `Integer.MIN_VALUE`